lib/osf-components/addon/components/node-metadata-form/template.hbs

```diff
@@ -331,14 +331,13 @@
             </dl>
         {{/if}}
     </FormControls>
-    {{#if @manager.node.isRegistration}}
+    {{#unless @manager.node.isAnonymous}}
```

Comment:
Is this true? Why would we not show subjects if the node is being viewed through an anonymous view only link?